src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -414,6 +415,27 @@ object ItemFeatures {
                 }"
             })
         }
+
+        if (Skytils.config.showItemQuality && extraAttr != null) {
+            val boost = extraAttr.getInteger("baseStatBoostPercentage")
+            val tier = extraAttr.getInteger("item_tier")
+
+            if (boost > 0 && tier > 0) {
+                val isMasterMode = requirementPattern.matchEntire(extraAttr.getString("dungeon_skill_req"))?.let {
+                    val level = it.groups["level"]?.value?.toIntOrNull() ?: return@let null
+                    level > 24
+                }
+
+                val floor = when (isMasterMode) {
+                    true -> "§4M${tier - 3}"
+                    false -> "§aF$tier"
+                    null -> "§b$tier"
```

Comment:
What is this case for?

Reply:
If they change the dungeon_skill_req String the Regex won't match (Or if the tag is not there at all, tho idk if that can happen) and with that you don't know whether it came from master mode or not so just default to the number representation

Reply:
Ah fair enough, I would prefer the case to be `else` instead just to make clear that it will catch all other values (even though a `Boolean?` has 3 states)